internal/events/trigger/trigger_event.go

```diff
@@ -257,7 +257,8 @@ https://dev.twitch.tv/docs/eventsub/handling-webhook-events#processing-an-event`
 
 		// Error checking for everything else
 		if reply.ResponseCode == 0 { // Zero will always be success
-			color.New().Add(color.FgGreen).Println(fmt.Sprintf(`✔ Forwarded for use in mock EventSub WebSocket server`))
+			fowardToEvtSubWS := `✔ Forwarded for use in mock EventSub WebSocket server`
```

Comment:
This variable is unnecessary. It can just be a normal string put in the Println(...) call